company/okrs/2020_q1.md

```diff
@@ -21,9 +21,7 @@
 1. **VP Business: Build a Customer Success organization that supports and helps us exceed our ARR goals.**
```

Comment:
I'd like to confirm our CE decision before finalizing this section. It may be moved to another team or changed otherwise.